Translation corrections (#481)
07th-mod/higurashi-console-arcs: Update/omot_023.txt

			NULL, "\"That's a thing of the past! It has nothing to do with Satoko!!", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/220100222", 256, TRUE);
	OutputLine(NULL, "　あんたたちが勝手に沙都子を敵とみなしてるだけだろっ！！」",

> 			NULL, " You're the ones regarding Satoko as if she's your enemy all on your own!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " You're all selfishly looking at Satoko as if she's your enemy!!\"", Line_Normal);
```

//r…距離が離れている俺たちでさえ、全身の血が凍りつくほどの震えが走り、気が遠のくように視界が狭窄する。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…距離が離れている俺たちでさえ、全身の血が凍りつくほどの震えが走り、気が遠のくように視界が狭窄する。",

> 			NULL, "...Even from all the way over here, my whole body froze and began to shiver. My field of vision narrowed as though I were stupified.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Even from all the way over here, my whole body froze and began to shiver. My field of vision absentmindedly narrowed.", Line_Normal);
```

//r俺たちの覚悟も、ここで決まった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "俺たちの覚悟も、ここで決まった。",

> 			NULL, "We all resolved ourselves right here as well.", Line_Normal);

Reviewer: ```suggestion
			NULL, "We all decided right here. We were fully prepared.", Line_Normal);
```

			NULL, "\"That's right, Keiichi-kun!!", Line_WaitForInput);
	ModPlayVoiceLS(4, 2, "ps3/s14/02/220200098", 256, TRUE);
	OutputLine(NULL, "　レナもこうなったら、絶対に容赦しない！！」",

> 			NULL, " Rena won't show them any mercy either now that things have come to this!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Now that things have come to this, Rena won't show them any mercy either!!\"", Line_Normal);
```

				NULL, "\"Y-yeah... That's right...!", Line_WaitForInput);
		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4bc_rika1191", 256, TRUE);
		OutputLine(NULL, "　勝敗は時の常、なんて言葉は嘘っぱち！」",

> 				NULL, " Whoever said the chance of war is uncertain was an outright liar!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Anyone saying saying the chance of victory is uncertain is an outright liar!\"", Line_Normal);
```
